### PR TITLE
ci: build UBI-based FIPS images

### PR DESCRIPTION
**Pull request by aboguszewski-sumo**
ref: https://github.com/SumoLogic/sumologic-otel-collector/pull/1440#pullrequestreview-1888740221

### CODE REVIEW

**Comment by aboguszewski-sumo** [comment]
build tags can be seen here: https://gallery.ecr.aws/sumologic/sumologic-otel-collector-dev

File: .github/workflows/release_builds.yml [hunk 1]
```diff
@@ -417,6 +426,13 @@ jobs:
             PLATFORMS="linux/amd64" \
             BUILD_TYPE_SUFFIX="-fips"
 
+      - name: Push joint UBI-based FIPS container manifest for all platforms to Open Source ECR
+        run: |
+          make push-container-manifest \
+            BUILD_TAG=${{ steps.extract_tag.outputs.tag }} \
+            PLATFORMS="linux/amd64"
+            BUILD_TYPE_SUFFIX="-ubi-fips"
```
Comment (sumo-drosiek) [inline .github/workflows/release_builds.yml:434]: I don't know if better is `-ubi-fips` or `-fips-ubi`

Comment (aboguszewski-sumo) [reply .github/workflows/release_builds.yml:434]: I used `-ubi-fips` as for shortcut of `UBI-based FIPS image`.

Comment (astencel-sumo) [reply .github/workflows/release_builds.yml:434]: I vote for `ubi-fips`

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**